maltd-frontend/src/components/composite/UserSearch/UserSearch.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable react/jsx-filename-extension */
 import React from "react";
 import PropTypes from "prop-types";
-import { Container, Row, Col, Spinner } from "reactstrap";
+import { Container, Row, Col } from "reactstrap";
```

Comment:
I'm okay not having the spinner here -> just wondering what the reasoning was behind removing it. Is it a requirement of the design or just a UI look/feel decision?

Reply:
There will have to be some test fixes if the Spinner is removed as well as the composite user search component has a test that checks for the existence of that component

Reply:
The approved mock-ups don't include such screen, thus it's not a hard requirement but a UI look/feel decision. We can display either the text indicator or the spinner, the latter imported from Reactstrap doesn't really align with our current style to me, what do you think?

Reply:
Sounds good, thanks!